src/Umbraco.Web.UI.Client/src/less/components/application/umb-drawer.less

```diff
@@ -7,50 +7,46 @@
     z-index: 10;
     width: @drawerWidth;
     background: @brownGrayLight;
-    box-shadow: inset 5px 0 20px rgba(0,0,0,.3);
-}
+    box-shadow: inset 5px 0 20px rgba(0, 0, 0, .3);
 
-.umb-drawer-view {
-    display: flex;
-    flex-direction: column;
-    height: 100%;
-    overflow: hidden;
+    &-view {
```

Comment:
I don't like this. This is going to make it a lot harder to find .umb-drawer-view in the sourcecode.

Reply:
Yup, fair call - I may have gone a bit heavy-handed. Have changed this to have separate classes for .umb-drawer-view, .umb-drawer-content and .umb-drawer-footer, to be grep-friendly. 

Makes better sense to split on the BEM modifier:
```
 .umb-group-panel {
    // stuff
    &__header { 
        // more stuff
    }
}
```

I still feel though there's benefit to embracing a degree of nesting in the LESS files, particularly as the project structure improves (ie component per file, sensibly namespaced etc).